custom_components/tuya_local/devices/wistar_roller_blind_nopos.yaml

```diff
@@ -22,7 +22,15 @@ primary_entity:
         min: 0
         max: 100
       mapping:
-        - invert: true
+        - invert: false
```

Comment:
If this does not need inverting, then it is the first device encountered that doesn't.  Normally Tuya uses the opposite definition of 0 and 100 than Home Assistant does.